docs: schema id docs (#8484)
confluentinc/ksql: docs/developer-guide/ksqldb-reference/create-table.md

ksqlDB can integrate with [Confluent Schema Registry](https://docs.confluent.io/current/schema-registry/index.html).
ksqlDB can use [Schema Inference](/operate-and-deploy/schema-registry-integration/#schema-inference) to
spare you from defining columns manually in your `CREATE TABLE` statements.

> ksqlDB can also use [Schema Inference With ID](/operate-and-deploy/schema-inference-with-id) to spare you

Reviewer: ```suggestion
ksqlDB can also use [Schema Inference With ID](/operate-and-deploy/schema-inference-with-id) to define

```

> from defining columns manually as well as enable you using physical schema to for data serialization.

Reviewer: ```suggestion
columns automatically and enable using physical schema for data serialization.

```
